### ISSUE TITLE
KeyError at /admin/fluent_pages/page/  "<class 'fluent_pages.pagetypes.textfile.models.TextFile'>"

### ISSUE THREAD

**Issue by bashu** [django-fluent/django-fluent-pages#132]
Running `example` project and trying to add plain text page:

```python
Traceback (most recent call last):
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/core/handlers/exception.py", line 42, in inner
    response = get_response(request)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/core/handlers/base.py", line 249, in _legacy_get_response
    response = self._get_response(request)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/core/handlers/base.py", line 217, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/core/handlers/base.py", line 215, in _get_response
    response = response.render()
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/response.py", line 109, in render
    self.content = self.rendered_content
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/response.py", line 86, in rendered_content
    content = template.render(context, self._request)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/backends/django.py", line 66, in render
    return self.template.render(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/base.py", line 208, in render
    return self._render(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/base.py", line 199, in _render
    return self.nodelist.render(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/base.py", line 994, in render
    bit = node.render_annotated(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/base.py", line 961, in render_annotated
    return self.render(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/loader_tags.py", line 174, in render
    return compiled_parent._render(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/base.py", line 199, in _render
    return self.nodelist.render(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/base.py", line 994, in render
    bit = node.render_annotated(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/base.py", line 961, in render_annotated
    return self.render(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/loader_tags.py", line 174, in render
    return compiled_parent._render(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/base.py", line 199, in _render
    return self.nodelist.render(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/base.py", line 994, in render
    bit = node.render_annotated(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/base.py", line 961, in render_annotated
    return self.render(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/loader_tags.py", line 174, in render
    return compiled_parent._render(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/base.py", line 199, in _render
    return self.nodelist.render(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/base.py", line 994, in render
    bit = node.render_annotated(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/base.py", line 961, in render_annotated
    return self.render(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/loader_tags.py", line 174, in render
    return compiled_parent._render(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/base.py", line 199, in _render
    return self.nodelist.render(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/base.py", line 994, in render
    bit = node.render_annotated(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/base.py", line 961, in render_annotated
    return self.render(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/loader_tags.py", line 174, in render
    return compiled_parent._render(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/base.py", line 199, in _render
    return self.nodelist.render(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/base.py", line 994, in render
    bit = node.render_annotated(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/base.py", line 961, in render_annotated
    return self.render(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/loader_tags.py", line 70, in render
    result = block.nodelist.render(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/base.py", line 994, in render
    bit = node.render_annotated(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/base.py", line 961, in render_annotated
    return self.render(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/loader_tags.py", line 70, in render
    result = block.nodelist.render(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/base.py", line 994, in render
    bit = node.render_annotated(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/base.py", line 961, in render_annotated
    return self.render(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/loader_tags.py", line 70, in render
    result = block.nodelist.render(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/base.py", line 994, in render
    bit = node.render_annotated(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/base.py", line 961, in render_annotated
    return self.render(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/tag_parser/basetags.py", line 176, in render
    return self.render_tag(context, *tag_args, **tag_kwargs)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/tag_parser/basetags.py", line 261, in render_tag
    return t.render(new_context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/base.py", line 210, in render
    return self._render(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/base.py", line 199, in _render
    return self.nodelist.render(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/base.py", line 994, in render
    bit = node.render_annotated(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/base.py", line 961, in render_annotated
    return self.render(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/polymorphic_tree/templatetags/polymorphic_tree_admin_tags.py", line 76, in render
    bits = [self._render_node(context, cl, node) for node in roots]
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/polymorphic_tree/templatetags/polymorphic_tree_admin_tags.py", line 68, in _render_node
    rendered = self.template_nodes.render(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/base.py", line 994, in render
    bit = node.render_annotated(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/base.py", line 961, in render_annotated
    return self.render(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/defaulttags.py", line 510, in render
    return self.nodelist.render(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/base.py", line 994, in render
    bit = node.render_annotated(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/base.py", line 961, in render_annotated
    return self.render(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/base.py", line 1044, in render
    output = self.filter_expression.resolve(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/base.py", line 711, in resolve
    obj = self.var.resolve(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/base.py", line 852, in resolve
    value = self._resolve_lookup(context)
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/django/template/base.py", line 915, in _resolve_lookup
    current = current()
  File "/Users/bashu/.virtualenvs/django-fluent-pages/lib/python2.7/site-packages/polymorphic_tree/models.py", line 137, in get_child_types
    return child_types[key]
KeyError: "<class 'fluent_pages.pagetypes.textfile.models.TextFile'>"
```

**Comment by vdboor** [comment]
This is now fixed by django-polymorphic-tree 1.5!

**Comment by bashu** [comment]
@vdboor thanks a lot!
